src/components/cards/CardHoverText.tsx

```diff
@@ -1,30 +0,0 @@
-import React, { ReactNode } from "react";
-import { BaseResourceType, Resources } from "../../types";
-
-interface CardHoverTextProps {
-  lockedOrUnlockedUnits?: "locked" | "unlocked";
```

Comment:
I noticed `"locked" | "unlocked"` has been used a lot....might be a good idea to break it out to its own type (or even enum)